pkg/kubelet/kubelet_node_status_test.go

```diff
@@ -118,6 +122,18 @@ func applyNodeStatusPatch(originalNode *v1.Node, patch []byte) (*v1.Node, error)
 	return updatedNode, nil
 }
 
+func notImplemented(action core.Action) (bool, runtime.Object, error) {
+	return true, nil, fmt.Errorf("no reaction implemented for %s", action)
+}
+
+func addNotImplatedReaction(kubeClient *fake.Clientset) {
+	if kubeClient == nil {
+		return
+	}
+
+	kubeClient.AddReactor("*", "*", notImplemented)
+}
```

Comment:
Sorry for the confusion, I didn't mean these specific methods so much as all the test setup boilerplate. Especilally the mock cAdvisor, which is unrelated to the function being tested.

Reply:
Can we handle this in a separate PR? The `ImagesFSInfo` and `RootFSInfo` are different; it takes some time to go through all case to have a helper func for it.